src/migratus/database.clj

```diff
@@ -188,24 +205,24 @@
   [db modify-sql-fn table-name]
   (log/info "creating migration table" (str "'" table-name "'"))
   (let [timestamp-column-type (datetime-backend? db)]
-    (sql/with-db-transaction [t-con db]
-      (sql/db-do-commands
-        t-con
-        (modify-sql-fn
-          (sql/create-table-ddl table-name [[:id "BIGINT" "UNIQUE" "NOT NULL"]
-                                            [:applied timestamp-column-type "" ""]
-                                            [:description "VARCHAR(1024)" "" ""]]))))))
+    (jdbc/with-transaction [t-con (connection-or-spec db)]
+      (sql-mig/do-commands
+       t-con
+       (modify-sql-fn
+        (str "CREATE TABLE " table-name
```

Comment:
Since table name can be supplied externally, I think it would be better to parametarize this to avoid potential for injection.

Reply:
now, `init-schema!` that calls `create-migration-table!` does not have access to the config so the API for that will have to change, or we can consider table-name is already escaped by `migration-table-name` function. 

```
(init-schema! @connection
                  (migration-table-name config)
                  (sql-mig/wrap-modify-sql-fn (:modify-sql-fn config)))

(defn migration-table-name [config]
  (:migration-table-name config default-migrations-table))
```

```
;; this proposed solution needs more work to make tests pass : 
(defn migration-table-name 
  "Construct table name and quote it.
  Users can choose quoting function via :table-quote-fn config key.
  Default quoting is next.jdbc.quoted/ansi"
  [config]
  (let [quote-table (or (:table-quote-fn config) q/ansi)
        table-name (:migration-table-name config default-migrations-table)]
    (quote-table table-name)))
```

UPDATE: It seems the situation is more complex: 

> [Quotes names in H2 are case sensitive](http://h2database.com/html/grammar.html#quoted_name), as required by the SQL specification. That means this will work:

http://h2database.com/html/grammar.html#quoted_name
https://stackoverflow.com/questions/10789994/make-h2-treat-quoted-name-and-unquoted-name-as-the-same

Reply:
I think we should do nothing right now, perhaps document the possibility of SQL injection. 
The migration table name is something the app developers choose so it's not that "user" facing.